EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/EventGenerator.java

```diff
@@ -57,10 +45,10 @@
      *
      * @param eventCount number of events to be generated
      * @param eventType type the generated events should be of
-     * @param attributeSchemas map containing all eventType-attributes and their user-defined, possible values
+     * @param attributeUserInput
      */
-    public void generateEvents(int eventCount, EapEventType eventType, Map<TypeTreeNode, String> attributeSchemas) {
-        generateEvents(eventCount, this.replayScaleFactor, eventType, attributeSchemas, getCurrentTimestamp());
+    void generateEvents(int eventCount, EapEventType eventType, List<AttributeInput> attributeUserInput) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Expected @param tag for 'attributeUserInput'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AJavadocMethod_BP2016)